loadBalancer/src/main/scala/com/github/pandafolks/panda/loadbalancer/ConsistentHashingState.scala

```diff
@@ -57,4 +71,10 @@ final class ConsistentHashingState(private val positionsPerIdentifier: Int = 100
 
   override def notifyAboutRemove(items: Iterable[Participant]): Task[Unit] =
     Task.parTraverseUnordered(items)(item => Task.eval(remove(item))).void
+
+  def clearEmptyGroups(): Task[Unit] =
```

Comment:
make it private - it should not be triggered from outside. In the tests you can use `invokePrivate` mechanism